src/videojs.ads.js

```diff
@@ -113,12 +114,14 @@ var
           updateEnded = function() {
             ended = true;
           };
-        player.currentTime(snapshot.currentTime);
```

Comment:
Reporting duration can be trickier with MSE or segmented file formats since accurate duration information is only available once a good portion of the video has been buffered. That has been the case anytime we play HLS though and this is the first time I've heard of this. Is it a bug in the new version of contrib-hls, a bug/fact-of-life with MSE, or reasonable behavior that contrib-ads needs to account for?